docs/source/user-manual/blocks.md

```diff
@@ -565,3 +565,33 @@ Preview Image URL
 
 Alignment
 : Options for alignment include left, right, center, and full width.
+
+## Autosave
+
+Volto provides an **autosave** mechanism that automatically captures your ongoing edits as you work on a page or content item.
+
+Changes made in the blocks editor are periodically saved in the background to prevent data loss, even if the user navigates away or the browser unexpectedly closes.
+
+`````{note}
+Autosave stores changes locally in your browser using `localStorage`. These changes remain available until you explicitly click the **Save** button, at which point they are persisted to the server and the local storage is cleared.
+
+Watch the video below for a quick demonstration.
+
+````{only} not text
+```{video} ../_static/user-manual/blocks/autosave-local.mp4
+:alt: Autosave
+`````
```

Comment:
Let's simplify the markup. A note isn't needed here, as it continues the previous paragraph explanation of what is going on and how to use it.

Also copy `packages/volto/src/icons/save.svg` to `docs/source/_static/user-manual/blocks/save.svg`

````suggestion
Autosave stores changes locally in your browser using [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
These changes remain available until you explicitly click the {guilabel}`Save` button <img alt="Save icon" src="../_static/user-manual/blocks/save.svg" class="inline">, at which point they are persisted to the server and the local storage is cleared.

Watch the video below for a quick demonstration.

```{video} ../_static/user-manual/blocks/autosave-local.mp4
:alt: Autosave
```
````